Services/TMS/Booking/classes/ilTMSBookingGUI.php

```diff
@@ -75,22 +77,46 @@ public function executeCommand() {
 		$crs_ref_id = (int)$_GET["crs_ref_id"];
 		$usr_id = (int)$_GET["usr_id"];
 
+		$gui_bindings = new Booking\GUIBindings
+			( $this->g_lng
+			, $this->g_ctrl
+			, $this->parent_gui
+			, $this->parent_cmd
+			, $this->getPlayerTitle()
+			, $this->getConfirmButtonLabel()
+			, $this->getOverViewDescription()
+			);
+
 		if((int)$this->g_user->getId() !== $usr_id && !$this->checkIsSuperiorEmployeeBelowCurrent($usr_id)) {
-			$this->redirectToPreviousLocation(array($this->g_lng->txt("no_permissions_to_book")), false);
+			$this->setParameter(null, null);
+			return $gui_bindings->redirectToPreviousLocation(array($this->g_lng->txt("no_permissions_to_book")), false);
```

Comment:
Hm ich hätte jetzt in den bindings eher nur Dinge erwartet, die sich mit der Darstellung beschäftigen. Sprachvariable, unterschiedliche Formelemente oder so.

Redirect ist doch eher ne Art Tool um von A nach B zu kommen.

Reply:
Fair enough. Für mich war `GUIBindings` die Bindings an ILIAS-GUI-Functionalitäten, zu denen Redirect imo schon gehört. Anderer Namensvorschlag?

Reply:
In ILIAS ist das eigentlich auch eine Funktion die von den GUI's benutzt wird. Das ganze wird erst zur Funktionalität indem man ilCtrl nur in GUI's verwenden darf.
Deshalb gibt es ja noch das Schlupfloch in den ilUtil mit denen man auch redirecten kann.

Ich würde das einfach aus den Bindings nehmen und in den Wizard verschieben. Die parents sollten ja auch da bekannt sein.